### DISCUSSION TITLE
Pocketbase on Oracle Ampere (ARM) cloud

### DISCUSSION THREAD

**Discussion by raf2001** [pocketbase/pocketbase#764] [Q&A] [answered]
Hi, 
Is there any possibility that Pocketbase to run on Oracle Ampere (ARM) cloud with ubuntu 22.04 installed on it?
It's arm infrastructure so maybe PB is not compatible with this type of machine?

**Comment by ganigeorgiev** [comment, marked as answer]
Based on the information from https://www.oracle.com/cloud/compute/arm/what-is-arm/, it seems that armv8 is used, so I guess the arm64 bundle [pocketbase_0.7.9_linux_arm64.zip](https://github.com/pocketbase/pocketbase/releases/download/v0.7.9/pocketbase_0.7.9_linux_arm64.zip) should work.

But please note that recently another user reported an issue on Raspberry Pi4 with it in https://github.com/pocketbase/pocketbase/issues/757 suspecting Go tooling regression so I'm not really sure (_I'll try to find some time in the future to test it on arm host somewhere_).

**Comment by 20162026** [comment]
can confirm that [pocketbase_0.7.9_linux_arm64.zip](https://github.com/pocketbase/pocketbase/releases/download/v0.7.9/pocketbase_0.7.9_linux_arm64.zip) works on oracle ampere, although I'm using ubuntu 20.04 on it (personal reasons)

**Reply by raf2001** [reply]
Yes and no...I have problem with accessing it : 
2022/10/12 22:11:24 listen tcp 130.61.x.x:443: bind: cannot assign requested address
but it seems that app is working - I don't have any other issue (probably network/configuration problem).
I'm using clean ubuntu 22.04 installation and have such problem.
I'll inform you if I'll resolve that issue

**Reply by ganigeorgiev** [reply]
@raf2001 This is not related to the binary. As long as you are not getting "cannot execute binary file: Exec format error" the binary should work just fine. This is probably a network misconfiguration (eg. missing a loopback address or similar).

**Reply by raf2001** [reply]
now I can confirm that with nginx proxy it's working.

**Comment by adampatterson** [comment]
I am running Ubuntu 22.04 on the Oracle Ampere (ARM) cloud and **linux_arm64.zip** also worked for me.
